### PR TITLE
[excel] (Workbook) Adding platform support note for insert worksheets

### PR DESCRIPTION
**Pull request by AlexJerabek**
Documentation for this issue: https://github.com/OfficeDev/office-js/issues/781

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f0512e1](https://github.com/OfficeDev/office-js-docs-pr/commits/f0512e124e54455b636525794075276553f5f0a3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJerabek-patch-1/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910211623463683-1377%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=pmxi374aIUtdPXqOUSaXTvdAOvL%2Bua0NgjR1d4IkoKg%3D&st=2019-10-21T16%3A19%3A52Z&se=2019-11-21T16%3A24%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/excel-add-ins-workbooks.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ reader.readAsDataURL(myFile.files[0]);
 
 > [!NOTE]
 > The `WorksheetCollection.addFromBase64` method is currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/excel-add-ins-workbooks.md]: I suggest you merge the two statements.

```suggestion
> The `WorksheetCollection.addFromBase64` method is currently only available in public preview for Office on Windows and Mac. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9f8cb83](https://github.com/OfficeDev/office-js-docs-pr/commits/9f8cb838df5a3732c3747c50886972590f876f2d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJerabek-patch-1/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910211922578289-1377%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yGkHR5tHygzvyZ4B088yELX%2FdPfOg4bJUOWHetBj4uc%3D&st=2019-10-21T19%3A19%3A06Z&se=2019-11-21T19%3A24%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. I clarified the text a bit more to indicate it's both in preview and only for Windows/Mac.

**Pull request merged**